morpheus/modules/schemas/examples/llm/content_extractor_schema.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
Do we want `examples` in the path here?

Reply:
I can see arguments in each direction. I'm somewhat for it because I don't like relative imports. But its also a little strange. Do you have any strong feelings here?

Reply:
No strong feelings either way, as long as it is intentional.